Concepts/Classes/Number.cs

```diff
@@ -1761,7 +1759,7 @@ public class BitsEnumerator : System.Collections.Generic.IEnumerator<bool>
 
             public BitsEnumerator(Bits bits, int start = 0, int bitSize = Bits.BitSize)
             {
-                if (bits == null) throw new ArgumentNullException(); Bits = bits;
```

Comment:
This is technically a breaking change but I am open to it.

Reply:
`Bits` is `struct`, so never `null`?

Reply:
This is correct, my apologies.